Gems/AWSCore/Code/Tools/ResourceMappingTool/resource_mapping_schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "type": "object",
+    "title": "O3DE AWS Resource mapping file schema",
+    "required": ["AWSResourceMappings", "AccountId", "Region", "Version"],
+    "properties": {
+        "AWSResourceMappings": {
+            "type": "object",
+            "title": "AWS resource mappings schema",
+            "patternProperties": {
+                "^.+$": {
+                    "type": "object",
+                    "title": "AWS resource entry schema",
+                    "required": ["Type", "Name/ID"],
+                    "properties": {
+                        "Type": {
+                            "$ref": "#/NonEmptyString"
+                        },
+                        "Name/ID": {
+                            "$ref": "#/NonEmptyString"
+                        },
+                        "AccountId": {
+                            "$ref": "#/AccountIdString"
+                        },
+                        "Region": {
+                            "$ref": "#/RegionString"
+                        }
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "AccountId": {
+            "$ref": "#/AccountIdString"
+        },
+        "Region": {
+            "$ref": "#/RegionString"
+        },
+        "Version": {
+            "pattern": "^[0-9]{1}.[0-9]{1}.[0-9]{1}$"
+        }
+    },
+    "AccountIdString": {
+        "type": "string",
+        "pattern": "^[0-9]{12}$|EMPTY|^$"
+    },
+    "NonEmptyString": {
+        "type": "string",
+        "minLength": 1
+    },
+    "RegionString": {
+        "type": "string",
+        "pattern": "^[a-z]{2}-[a-z]{4,9}-[0-9]{1}$"
+    },
+    "additionalProperties": false
+}
```

Comment:
Nit: Add a new line at the end of the file.

Reply:
done